nucleus/common/amx-core/src/main/java/org/glassfish/admin/amx/impl/mbean/LoggingImpl.java

```diff
@@ -411,7 +411,7 @@ private Properties attributesToProps(List<Attribute> attrs) {
     public Map<String, Number>[] getErrorInfo() {
         unimplemented();
 
-        final List<Map<String, Object>> infos = null; //getLogMBean().getErrorInformation();
```

Comment:
`null` would cause NPE in line 416 - except we never get there because `unimplemented()` above already throws an exception.